extensions/amp-vi-player/OWNERS

```diff
@@ -0,0 +1,10 @@
+// For an explanation of the OWNERS rules and syntax, see:
+// https://github.com/ampproject/amp-github-apps/blob/master/owners/OWNERS.example
+
+{
+  rules: [
+    {
+      owners: [{name: 'cosmin-vi'}, {name: 'ampproject/wg-ui-and-a11y'}],
```

Comment:
Can you please update `ampproject/wg-ui-and-a11y` to `ampproject/wg-components`.

Reply:
Done. Thank you.